### PR TITLE
add caption to dno slide

### PR DESCRIPTION
**Pull request by denisnoel**
Hey @duviviereib can you please review?

### CODE REVIEW

File: _posts/0000-01-15-denisnoel.md [hunk 1]
```diff
@@ -7,3 +7,4 @@ title: "Welcome to our slide deck!"
 {: .center}
 
 Caption here
+DJ Slayer
```
Comment (denisnoel) [inline _posts/0000-01-15-denisnoel.md:10]: ok

**Review by denisnoel — COMMENTED** [review]
ok

File: _posts/0000-01-15-denisnoel.md (hunk 1 above)
Comment (arnaudolig) [reply _posts/0000-01-15-denisnoel.md:10]: Superb job!

**Review by eib-fabianburtard — APPROVED** [review]
Wonderful 👍

**Pull request merged**